### ISSUE TITLE
TheLounge Install not working correctly with The Lounge 3.0

### ISSUE THREAD

**Issue by theoneandonlyewok** [liaralabs/swizzin#182]
Not sure what changed. 

Every time I try to use the box command to install TheLounge it gives this error now:
```
Installing lounge
/usr/local/bin/swizzin/install/lounge.sh: line 475: /home/lounge/.lounge/users/theoneandonlyewok.json: No such file or directory
```

Nothing is ever configured, but TheLounge is installed. You can get to the login screen from the correct URL but can't add any users or anything else.

I tried using the "TheLounge" commands to see if i could manually add a user but everything just errors out.
```

thelounge list
2019-02-10 02:04:30 [ERROR] /root/.thelounge/users does not exist. 
 thelounge add theoneandonlyewok
2019-02-10 02:06:17 [ERROR] /root/.thelounge/users does not exist.
```

**Comment by theoneandonlyewok** [comment]
Looks like directory changed from .lounge to .thelounge with newest version of TheLounge.
Install script still uses .lounge.

**Comment by liaralabs** [comment]
Thanks for the heads up.

Install issues should be fixed as of 5785a916923fa84afb1e8c3db4ecd9fb4f4ed05c
